src/panels/MissionsPanel/util.ts

```diff
@@ -0,0 +1,42 @@
+import { isDateValid } from '@/redux/time/util';
+
+export function jumpToTime(
+  currentTime: number | undefined,
+  time: string,
+  luaApi: OpenSpace.openspace | null,
+  fadeTime: number = 1
+): void {
+  if (currentTime === undefined) {
+    return;
+  }
+
+  const utcDate = new Date(time);
+
+  if (!isDateValid(utcDate)) {
+    return;
```

Comment:
Just a thought: We should maybe start distinguishing between which of our early outs are "valid" ones, and which are not. For those that are not, we should throw an error with a meaningful message instead.  

Something to consider